grade_test.go

```diff
@@ -235,21 +239,25 @@ func TestCompileWallFailure(t *testing.T) {
 }
 
 func TestRunCompiled(t *testing.T) {
-	cpp := `#include <iostream>
-	int main() { std::cout << "Hello world!" << std::endl; }`
+	cpp := `
+	#include <iostream>
+	using namespace std;
+		int main(int argc, char** argv) {
+    	cout << "Hello, C Plus Plus!" << endl;
+    	return 0;
+	}
```

Comment:
may omit `int argc, char** argv` when not used, use consistent formatting

Reply:
is there any particular reason for this change?

Reply:
I honestly couldn't think of a valid reason now that I think about it. Going to change back to what it was before.